### ISSUE TITLE
BC_K8S_14/CKV_K8S_15 "Image Pull Policy should be Always" should not be reported when using a digest

### ISSUE THREAD

**Issue by candrews** [bridgecrewio/checkov#4776]
**Describe the issue**
[BC_K8S_14/CKV_K8S_15 "Image Pull Policy should be Always"](https://docs.bridgecrew.io/docs/bc_k8s_14) should not be reported when using a digest.

Per the documentation:
>  When the imagePullPolicy is set to Always, you ensure the latest version of the image is deployed every time the pod is started.

By specifying the digest, the same image is always used, so there is no concern about the latest version (since the the version cannot change), negating the need to always pull the image.

**Examples**
Save the following as `deployment.yaml`:
```yaml
apiVersion: v1
kind: Pod
metadata:
  name: <name>
spec:
  containers:
    - name: nginx
      image: 'nginx@sha256:b0ad43f7ee5edbc0effbc14645ae7055e21bc1973aee5150745632a24a752661'
      imagePullPolicy: IfNotPresent
```
Run: `checkov --framework kubernetes -f deployment.yaml -c CKV_K8S_15`

Actual result:
FAILED

Expected result:
Passed

**Version (please complete the following information):**
 - Checkov Version 2.3.126
